Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+/// A view that displays a text input field with a prompt text message for a user.
+struct TextEditorWithPlaceholder: View {
+    // MARK: - Private interface
+
+    private let emptyTextEditorOpacity = 0.25
+    private let filledTextEditorOpacity = 1.0
+
+    // MARK: - Internal interface
+
+    /// The text to be shown in the text input field.
```

Comment:
nit:

Is that also a text to edit? Should we update the doc comment?

Reply:
Yes, sure. Thanks, fixed it.